core/field_dropdown.js

```diff
@@ -101,7 +102,12 @@ Blockly.FieldDropdown.prototype.init = function() {
  * @private
  */
 Blockly.FieldDropdown.prototype.showEditor_ = function() {
-  Blockly.WidgetDiv.show(this, this.sourceBlock_.RTL, null);
+  // If there is an existing drop-down we own, this is a request to hide the drop-down.
```

Comment:
I don't understand this comment.  Or rather, I think I don't understand the flow here.  Why would showEditor_ actually mean hideEditor and return?